src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery userFirst = new Lottery();
+        Lottery userSecond = new Lottery();
+        Lottery userThird = new Lottery();
+        System.out.println(userFirst.getRandomBall().toString());
+        System.out.println(userSecond.getRandomBall().toString());
+        System.out.println(userThird.getRandomBall().toString());
```

Comment:
please extract Lottery to class variable, we do not need to have 3 instances
also please use cycle to get balls